example_test.py

```diff
@@ -72,12 +73,6 @@ def test_create_file(self):
 
     def test_delete_file(self):
         """Test example.delete_file()
-
-        `self.fs.CreateFile()` is convenient because it automatically creates
```

Comment:
Add to the module docstring at the top:

Fake filesystem functions like `CreateFile()`, `CreateDirectory()`, `CreateLink()` are often used to set up file structures at the beginning of a test.  You could also use the familiar `open()`, `os.mkdirs()` and so on.  `CreateFile()` is particularly convenient because it creates all parent directories and allows you to specify the contents and size of the file.